trace-sdk/src/main/java/io/bitrise/trace/TraceSdk.java

```diff
@@ -69,11 +74,21 @@ public synchronized static void init(@NonNull final Context context) {
             initDataCollection(context);
             initLifeCycleListener(context);
             initNetworkTracing();
+            TraceLog.i(String.format(LogMessageConstants.TRACE_DEBUG_FLAG_STATUS, DEBUG_ENABLED));
         } else {
             TraceLog.e(new TraceException.TraceConfigNotInitialisedException());
         }
     }
 
+    /**
+     * Flag to enable TraceSdk into debug mode.
```

Comment:
Could you please add more information on what does this debug mode mean? Developers might have questions what does this explicitly do.

Reply:
Have added a sentence on what it adds currently - ` Currently, this will add more debug log messages.` - i've deliberately left it light on exactly what kind of debug messages because i can see it being useful to add more in the future. Does that make sense?

Reply:
Sure, mentioning it adds logging is enough, did not mean to add a list of messages, just what differences it has with the release mode.